orcid-persistence/src/main/resources/db/updates/create_profile_view.xml

```diff
@@ -0,0 +1,14 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="CREATE-PROFILE-VIEW" author="Daniel Palafox">
+        <createView viewName="profile_view">
```

Comment:
@DanielPalafox view name should be dw_profile

Reply:
@amontenegro done!